<launch>

  <node name="generate_command_joy"   pkg="kondo_b3mservo_rosdriver" type="generate_command_joy.py" output="screen"/>

  <node name="torque_control" pkg="kondo_b3mservo_rosdriver" type="torque_control.py" output="screen">
    <param name="servo_id" value="4"/>
  </node>

  <node name="joy_node"   pkg="joy" type="joy_node">
    <param name="autorepeat_rate" value="10"/>
  </node>

</launch>
